cozy/syntax.py

```diff
@@ -22,6 +22,12 @@ class Type(ADT): pass
 TString             = declare_case(Type, "TString")
 TNative             = declare_case(Type, "TNative", ["name"])
 THandle             = declare_case(Type, "THandle", ["statevar", "value_type"])
+## TODO: I find the very short field names "t" confusing; there are a lot of
+## `t.t` in the code that are unnecessarily hard to understand.
+## I would use something like "eltType" instead, or "elem_type" as is used
+## for heaps (for consistency with them).
+## This will also prevent the use of the identical field name "t" for
+## a different meaning in TApp than in TBag, TSet, and TList.
```

Comment:
I agree completely. When I wrote the code for heaps (which is much newer) I used `elem_type` for exactly this reason. Doing this refactoring will be a little tricky on untyped Python and it makes me nervous because it will be very easy to miss places. Thus I do not intend to address that problem in the short term; we should open a separate issue for it.